device/samples/pnp/pnpTemperatureController.js

```diff
@@ -11,6 +11,12 @@ const Message = require('azure-iot-device').Message;
 //  'HostName=<iothub_host_name>;DeviceId=<device_id>;SharedAccessKey=<device_key>'
 const deviceConnectionString = process.env.IOTHUB_DEVICE_CONNECTION_STRING;
 
+// DPS connection information
+var provisioningHost = process.env.PROVISIONING_HOST;
+var idScope = process.env.PROVISIONING_IDSCOPE;
+var registrationId = process.env.PROVISIONING_REGISTRATION_ID;
+var symmetricKey = process.env.PROVISIONING_SYMMETRIC_KEY;
+
```

Comment:
you also need the environmental variable for device connection string that is gonna be used when dps not available